src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void writeToFile(String report, String filePath) {
```

Comment:
String report? This method can write to file reports only? I do not think so. Remember about abstraction.